frontend/components/TextInput/TextInput.css

```diff
@@ -9,4 +9,20 @@
 
 .input-in-label {
   background-color: inherit;
+}
+
+.resizable {
+  resize: both;
+}
+
+.not-resizable {
+  resize: none;
+}
+
+.full-height {
+  height: 100vh;
```

Comment:
Maybe this should account for the save bar e.g., ` calc(100vh - 135px)`?

Reply:
Sounds good, I'll add it